governance.md

```diff
@@ -11,6 +11,25 @@ https://github.com/pangeo-data/governance
 
 ### The Steering Council
 
+The Project will have a Steering Council that consists of Project Contributors who have produced contributions that are substantial in quality and quantity, and sustained over at least one year. The overall role of the Council is to ensure, through taking input from the Community, the long-term well-being of the project, both technically and as a community. As Pangeo draws its strength from representing the Community, it strives to avoid fragmentation of the comminity into separate projects, if possible.
```

Comment:
As Pangeo draws --> As The Project draws

Reply:
Some of this is duplicative of (and in conflict with) #1, which specifies criteria for membership on the council.